### PR TITLE
Frequency refills

### PR DESCRIPTION
**Pull request by mayerm94**
Add frequency and refills support to medications and substances to align with qdm spec.

Pull requests into Bonnie require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] This PR is into the correct branch.
- [x] JIRA ticket for this PR: https://jira.mitre.org/browse/BONNIE-1460
- [x] JIRA ticket links to this PR
- [x] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [x] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced. NA
- [x] Tests are included and test edge cases NA
- [x] Tests have been run locally and pass (remember to update Gemfile when applicable) NA
- [x] Test fixtures updated and documented as necessary ( see [internal wiki] (https://gitlab.mitre.org/bonnie/internal-documentation/wikis/testing#test-fixtures) ) NA
- [x] Code coverage has not gone down and all code touched or added is covered. 
     * In rare situations, this may not be possible or applicable to a PR. In those situations:
         1. Note why this could not be done or is not applicable here: 
         2. Add TODOs in the code noting that it requires a test
         3. Add a JIRA task to add the test and link it here: 

Branch | Back End Coverage | Front End Coverage
-- | -- | --
master | [(88.9% covered at 33.76 hits/line) | [Statements   : 57.76% ( 6968/12064 ) <br> Branches     : 40.9% ( 1839/4496 )<br> Functions    : 48.56% ( 1162/2393 ) <br>Lines        : 55.57% ( 6251/11248 )]
[New Branch] | [88.9% covered at 33.76  #hits/line)] | Statements   : 57.76% ( 6968/12064 ) <br>Branches     : 40.9% ( 1839/4496 )<br> Functions    : 48.56% ( 1162/2393 )<br> Lines        : 55.57% ( 6251/11248 )

- [x] Automated regression test(s) pass

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA test links: NA
- [x] Justification for using JIRA tests: NA
- [x] JIRA tests have been added to sprint: NA

**Reviewer 1:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree

**Reviewer 2:**

Name: @losborne
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree

### CODE REVIEW

**Comment by mayerm94** [comment]
Test with these branches.
https://github.com/projecttacoma/cql_qdm_patientapi/tree/frequency_refills
https://github.com/projectcypress/health-data-standards/tree/frequency_refills

**Review by lukoktonos — COMMENTED** [review]
Everything looks good, will wait for cql_qdm_patientapi to be merged before finalizing approval.

**Review by lukoktonos — DISMISSED** [review]

**Review by jbradl11 — DISMISSED** [review]
Changes make sense. Tests pass locally. Looks good.

**Review by hossenlopp — DISMISSED** [review]
I believe I'm a third reviewer. 👍

**Review by jbradl11 — APPROVED** [review]
Re-reviewing for updated Gemfile and Gemfile.lock file. Looks good.

File: Gemfile [hunk 1]
```diff
@@ -13,19 +13,19 @@ gem 'less-rails'
...
 gem "non-stupid-digest-assets"
 
-# gem 'health-data-standards', '~> 4.0'
-# gem 'cql_qdm_patientapi', '~> 1.0'
+gem 'health-data-standards', '~> 4.0.4'
+gem 'cql_qdm_patientapi', '~> 1.1'
 gem 'simplexml_parser', '~> 1.0'
 gem 'hqmf2js', '~> 1.4'
-# gem 'bonnie_bundler', '~> 2.0'
+gem 'bonnie_bundler', '~> 2.1.0'
```
Comment (c-monkey) [inline Gemfile:20]: Should we change this to '~> 2.1'?

Comment (c-monkey) [reply Gemfile:20]: If made this change locally, all tests still pass. I don't feel strongly either way.

**Review by c-monkey — APPROVED** [review]
Tests pass.

Gemfile changes work and look good. Lizzie and I discussed if we should have bumped HDS to 4.1 rather than 4.0.4, which we should have, but we're not going to hold up the release for that. We'll make a 4.1 gemspec push separately but not update dependent projects for now.

**Pull request merged**